test/check-keys

```diff
@@ -150,4 +150,98 @@ class TestKeys(MachineCase):
         self.allow_journal_messages('Missing callback called fullpath = /home/user/.ssh/authorized_keys')
         self.allow_journal_messages('')
 
+    def testPrivateKeys(self):
+        b = self.browser
+        m = self.machine
+
+        def list_keys():
+            return b.eval_js("require('base1/cockpit').spawn([ 'ssh-add', '-l' ])")
+
+        # Put all the keys in place
+        m.execute("mkdir -p /home/admin/.ssh")
+        m.upload([
+            "files/ssh/id_rsa",
+            "files/ssh/id_dsa",
+            "files/ssh/id_rsa.pub",
+            "files/ssh/id_dsa.pub"
+        ], "/home/admin/.ssh/")
+        m.execute("chmod 600 /home/admin/.ssh/*")
+        m.execute("chown -R admin:admin /home/admin/.ssh")
+
+        self.login_and_go()
+
+        id_rsa = "2048 SHA256:SRvBhCmkCEVnJ6ascVH0AoVEbS3nPbowZkNevJnXtgw"
+        id_dsa = "1024 SHA256:x6S6fxMuEyqhpwNRAIK7ms6bZDY6xK9wzdDr2kCaWVY"
+
+        keys = list_keys()
+        self.assertIn(id_rsa, keys)
```

Comment:
This won't pass on atomic with local-ssh. Should we adjust the test? or skip for atomic.